packages/common/amplify_db_common/test/web_test.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('browser')
+
+import 'dart:async';
+
+import 'package:amplify_core/amplify_core.dart';
+import 'package:amplify_db_common/src/connect.web.dart';
+import 'package:async/async.dart';
+import 'package:aws_common/testing.dart';
+import 'package:drift/drift.dart';
+import 'package:test/test.dart';
+
+import 'main_test.dart';
```

Comment:
I would add a common file with shared elements. Importing a file that's vm-only test into a web test is a bit weird.